public/css/mixin/progress-bar.less

```diff
@@ -9,23 +9,32 @@
 
     .above {
       padding: ~"calc(1em + 1px) 10%";
+
+      .now {
+        width: auto;
+        z-index: 2;
```

Comment:
Please check again here. I don't think that's correct or the only required change. This is related to the other z-index definitions and these you didn't touch. It's also related to
#44. The li's width is too wide atm or shouldn't be related to this "hover" behavior, but it blocks the hover at the moment on the right of `Now` if it's still very left.

Reply:
Disabling both in dev-tools has no effect. So I have removed the block.